packages/cdktf/lib/backends/artifactory-backend.ts

```diff
@@ -8,7 +8,9 @@ import {
   DataTerraformRemoteStateConfig,
 } from "../terraform-remote-state";
 
-// eslint-disable-next-line jsdoc/require-jsdoc
+/**
+ * @deprecated the artifactory backend has been deprecated in Terraform v1.2.3 and was removed in Terraform v1.3
```

Comment:
@ansgarm What if we did something like this? I was thinking maybe we should differentiate between our product and Terraform itself. But let me know if this feels like too much. If you like it, maybe we could do this for all of them?

```suggestion
 * @deprecated CDK for Terraform no longer supports the artifactory backend. Terraform deprecated artifactory in v1.2.3 and removed it in v1.3. 
```

Reply:
Sounds good! I like it 👍